test/new-e2e/tests/installer/host/host.go

```diff
@@ -161,6 +161,11 @@ func (h *Host) AssertPackageInstalledByInstaller(pkgs ...string) {
 	}
 }
 
+// AssertPackageVersion checks if a package is installed with the correct version
+func (h *Host) AssertPackageVersion(pkg string, version string) {
+	h.remote.MustExecute("ls /opt/datadog-packages/" + pkg + "/" + version)
+}
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** 
```suggestion
// AssertPackageVersion checks if a package is installed with the correct version
func (h *Host) AssertPackageVersion(pkg string, version string) {
	state := h.State()
	state.AssertDirExists(filepath.Join("/opt/datadog-packages/", pkg, version), 0755, "root", "root")
}

```

Reply:
Though tbh this is a nice change, we could use it in other tests too